aws-android-sdk-s3-test/src/androidTest/java/com/amazonaws/services/s3/CleanupBucketIntegrationTests.java

```diff
@@ -61,7 +61,11 @@ public void testCleanup() {
                     || bucket.getName().startsWith("s3-low-level-presigned-url")
                     || bucket.getName().startsWith("java-bucket-policy-integ-test")
                     || bucket.getName().startsWith("copy-object-integ-test")
-                    || bucket.getName().startsWith("java-server-side-encryption-integ-test")) {
+                    || bucket.getName().startsWith("java-server-side-encryption-integ-test")
+                    || bucket.getName().startsWith("acceleration-integration-test-")
+                    || bucket.getName().startsWith("amazon-transfer-util-integ-test")
+                    || bucket.getName().startsWith("java-bucket-lifecycle-integ-test-")
+                    || bucket.getName().startsWith("java-s3-object-iteration-test-")) {
```

Comment:
It's hard to look at this without thinking there's a better way to approach this such as a standard prefix or using S3 tags to indicate it's a test bucket. Thoughts?

Reply:
We can for some of these but interestingly a lot these patterns are not created by our integration tests. I guess these are being created by tests from other teams (not sure why?). So we might have trace their origin and make a fix accordingly.

Reply:
We're sharing the account with other processes? If so, let's fix that immediately to make this easier.